plugins/inputs/graylog/graylog.go

```diff
@@ -34,6 +35,7 @@ type GrayLog struct {
 	Metrics  []string
 	Username string
 	Password string
+	Timeout  config.Duration
```

Comment:
Can you please add the `toml` tags to the struct fields that receive user-options like
```suggestion
	Timeout  config.Duration `toml:"timeout"`
```
This makes it more clear which fields are configured by the user...